server/database/handlers/notificationDBHandler.js

```diff
@@ -0,0 +1,77 @@
+const models = require("../models");
+const Notification = models.Notification;
+const Product = models.Products;
+const CronJob = require("cron").CronJob;
+const { scrapeAmazon, scrapeEbay } = require("../../scrapers/index");
+
+async function createNotification(
+  productId,
+  userId,
+  previousPrice,
+  currentPrice
+) {
+  const notification = await Notification.create({
+    productId: productId,
+    userId: userId,
+    previousPrice: previousPrice,
+    currentPrice: currentPrice,
+  })
+    .then((res) => {
+      return res;
+    })
+    .catch((err) => {
+      console.log(err);
+      console.log("Notification not created.");
+      return null;
+    });
+  return notification;
+}
+
+const getSalePrices = async function () {
+  const notifications = [];
+  await Product.findAll().then((products) => {
+    products.map((product) => {
+      const url = product.productURL;
+      const website = url.split(".")[1];
+      if (website === "amazon") {
+        scrapeAmazon(url).then((res) => {
+          if (res.salePrice) {
```

Comment:
Over here, I'd also update the `product.productPrice` too, just in case the price of the product ever changes.

This one's from experience, but the one shirt that I kept testing the ProductConfirmationDialog with changed it's price while I was testing it, and I was super confused why I was getting the wrong price until I refreshed the Amazon page.

Reply:
It doesn't need to be in an if statement, just something that always happens.